src/Symfony/Bridge/Doctrine/Security/RememberMe/DoctrineTokenProvider.php

```diff
@@ -136,6 +137,56 @@ public function createNewToken(PersistentTokenInterface $token)
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function verifyToken(PersistentTokenInterface $token, string $tokenValue)
+    {
+        if (hash_equals($token->getTokenValue(), $tokenValue)) {
+            return true;
+        }
+
+
+        $tmpSeries = preg_replace('{=+$}', '_', $token->getSeries());
+        // if we cannot generate a unique series it is not worth trying further
+        if ($tmpSeries === $token->getSeries()) {
+            return false;
+        }
+
+        try {
+            $tmpToken = $this->loadTokenBySeries($tmpSeries);
+        } catch (TokenNotFoundException $e) {
+            return false;
+        }
+
+        if ($tmpToken->getLastUsed()->getTimestamp() + 60 < time()) {
```

Comment:
Perhaps a coment why the `60` here? That we assume that the old token is invalid after 60 seconds, so for a whole minute it would be possible to use the old **and** the new `rememberMe`-token. 

If that is a critical issue we somehow need to make that configurable.